corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -561,11 +561,10 @@ def data_source_properties(self):
 
     def _get_data_source_properties_from_case(self, case_properties):
         property_map = {
-            'closed': _('Case Closed'),
-            'user_id': _('User ID Last Updating Case'),
-            'owner_name': _('Case Owner'),
-            'mobile worker': _('Mobile Worker Last Updating Case'),
-            'case_id': _('Case ID')
+            'user_id': _('user_id_last_updating_case'),
```

Comment:
These shouldn't be translated. You can remove `owner_name` and `case_id` from the list as you did with `closed`.